pkg/controller/volume/attach_detach_controller.go

```
@@ -442,7 +443,7 @@ func (adc *attachDetachController) getPVSpecFromCache(
 
 	// Do not return the object from the informer, since the store is shared it
 	// may be mutated by another consumer.
-	clonedPVObj, err := api.Scheme.DeepCopy(pv)
+	clonedPVObj, err := api.Scheme.DeepCopy(*pv)
 	if err != nil || clonedPVObj == nil {
 		return nil, fmt.Errorf("failed to deep copy %q PV object", name)
```

Add the error to this error message, we are swallowing it currently